sdk/src/main/java/io/radar/sdk/Radar.kt

```diff
@@ -533,9 +533,13 @@ object Radar {
         if (fraud) {
             RadarSettings.setSharing(this.context, false)
         }
+        // remove the previous RadarActivityLifecycleCallbacks so we only keep one.
```

Comment:
can you elaborate a little more on why this is needed?

Reply:
If initialize is called multiple times, we only keep one callback. I was able to do this with the back button for the example app. I'll move this out to a separate PR for further investigation